Mage/src/main/java/mage/abilities/keyword/MoreThanMeetsTheEyeAbility.java

```diff
@@ -22,7 +22,9 @@ public MoreThanMeetsTheEyeAbility(Card card, String manaCost) {
         this.newId();
 
         // getSecondFaceSpellAbility() already verified that second face exists
-        this.setCardName(card.getSecondCardFace().getName() + " with Disturb");
+        this.setCardName(card.getSecondCardFace().getName());
+        this.name = "Cast " + cardName + " using";
```

Comment:
So clearly Disturb isn't correct, but I don't quite understand how you're changing it, could you explain?

Reply:
I think that name is the "cast ability" full name in the menu / in automated tests.
Here are before (with Arcee):
![mtmte1](https://github.com/magefree/mage/assets/34709007/cea5f0e0-9919-4161-b
![mtmte2](https://github.com/magefree/mage/assets/34709007/4350d16f-9178-4050-9724-0b0faaa1bcbf)
4b9-1dc5df5384a4)
![mtmte3](https://github.com/magefree/mage/assets/34709007/a762a429-5d98-4266-a998-f08b47a1ac5c)

Oh there are logs message too I did miss:
![image](https://github.com/magefree/mage/assets/34709007/0980fa9f-42b3-4966-a829-bdf783947294)

And after:

![mtmte_after_1](https://github.com/magefree/mage/assets/34709007/b7f21ff4-d6f4-4a44-a23d-c058853d6228)
![mtmte_after_2](https://github.com/magefree/mage/assets/34709007/9ff98111-094a-4bfb-9d7c-c2c3bea86289)
![mtmte_after_log](https://github.com/magefree/mage/assets/34709007/ee5c0063-1666-4bfd-b841-29d5af5faac7)

I'll figure out how to have both the cast ability name and the log message look right.

Reply:
Ok what went wrong is that there was no dedicated SpellAbilityCastMode, and it was set directly and not using `SpellAbility->setSpellAbilityCastMode`

Cleaning all that for More Than Meets the Eye and Disturb.